### PR TITLE
adding Versions section to readme

### PR DESCRIPTION
**Pull request by SomkaPe**

### CODE REVIEW

**Comment by polita** [comment]
@weijjia The travis build failed. Is this expected?
@SomkaPe Once the travis issue is resolved, :shipit:

**Comment by SomkaPe** [comment]
@weijjia The travis build failed. Is this expected?  -> do I need to Authorize Travis CI to access my account ?

**Comment by heidijinxujia** [comment]
I restarted the checks. Travis usually fails at the first run and gets passed at the second run. Don't know why... @SomkaPe

File: README.md [hunk 1]
```diff
@@ -7,7 +7,10 @@ A Work Account is an identity you use to get work done no matter if at your busi
 
 ## ADAL for Android 1.10 Released!
 
-Recent version is 1.10.0
+## Versions
+Current version - 1.10.0  
+Minimum  recommended version - 1.1.16
```
Comment (weijjia) [inline README.md]: >   [](start = 7, length = 2)

nit: extra space between Minimum and recommended

Comment (weijjia) [inline README.md]: >   [](start = 37, length = 2)

also remove the trailing spaces here and the above line

**Comment by weijjia** [comment]
@polita @SomkaPe I think this is the memory issue on where the travis is running causing the dexing failure. If it fails next time, try to restart it. :shipit: when the comments around spaces are addressed.

@heidijinxujia @addev-ashish We should make increase the memory for gradle build setting if it fails quite often.

File: README.md (hunk 1 above)
Comment (SomkaPe) [inline README.md]: two trailing spaces is a part of Markdown syntax, used for line break

**Pull request merged**